.github/workflows/ci-build.yml

```diff
@@ -10,6 +10,34 @@ on:
       - master
 
 jobs:
+  test-with-coverage:
+    name: Test with Coverage
+    runs-on: ubuntu-latest
```

Comment:
As far as I remember we had issues with some dependencies in the latest Ubuntu and had to switch to some specific version. So maybe it's safer to stick with some LTS version (*.04 versions)? cc: @MatousJobanek

Reply:
oh, I did not know about that 😮 
yes, indeed, we could use Ubuntu 18.04 or 20.04: https://github.com/actions/virtual-environments#available-environments

Reply:
switched to Ubuntu 20.04 (https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) in b8109ce

Reply:
Yeah. 20.04 would be preferable: https://ubuntu.com/about/release-cycle

Reply:
it's not a big deal for these CI jobs - there was some problem with a specific version of Podman that was installed in the latest version of Ubuntu - that's why we use 18.04 in CD job

Reply:
apart from that - I guess that 20.04 & latest is the same: https://github.com/actions/virtual-environments#available-environments

Reply:
right, `20.04` and `latest` are equivalent. I'll go ahead and merge with `20.04` for now, but we can always change back to `latest` (or something else) in the future if needed